kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -7839,7 +7839,7 @@ messages:
             j = 1;
             while j <= Length(stuff)
             {
-               if (Nth(stuff,j+1) = $)
+               if (Nth(stuff,j+1) = $) OR Nth(stuff, j + 1) = RentableRoom_expiry_warning
```

Comment:
This sure looks random out of context.  Add a comment explaining why it's here, and the fact that it can be deleted after a certain date (say, October 2023).

Reply:
Done.